code/mission/missionparse.cpp

```diff
@@ -317,6 +317,7 @@ flag_def_list_new<Mission::Parse_Object_Flags> Parse_object_flags[] = {
     { "hide-in-mission-log",			Mission::Parse_Object_Flags::SF_Hide_mission_log,		true, false },
     { "same-arrival-warp-when-docked",		Mission::Parse_Object_Flags::SF_Same_arrival_warp_when_docked,		true, false },
     { "same-departure-warp-when-docked",	Mission::Parse_Object_Flags::SF_Same_departure_warp_when_docked,	true, false },
+	{ "attackable_if_no_collide",			Mission::Parse_Object_Flags::OF_Attackable_if_no_collide, true, false },
```

Comment:
It looks like there's been a tabs/spaces mixup that threw things out of whack, especially visible if I view the whole file in github: https://github.com/scp-fs2open/fs2open.github.com/blob/bc000b5f692be21c2f9303067e17bd78d6897dfa/code/mission/missionparse.cpp#L320